scripts/fast-start.sh

```diff
@@ -181,6 +189,9 @@ KEYNAME=$(grep keyname /root/AppScalefile | cut -f 2 -d ":")
 [ -z "${KEYNAME}" ] && { echo "Cannot discover keyname: is AppScale deployed?" ; exit 1 ; }
 
 # Deploy sample app.
+if [ -z "${ADMIN_EMAIL}" ]; then
```

Comment:
Actually this is better: if the email is not set, the next command will fail creating a bad experience. If it's empty it's becasue we set the test flag, so we have a@a.com.